internal/cephfs/snapshot.go

```diff
@@ -45,29 +45,16 @@ type cephfsSnapshot struct {
 	ReservedID string
 }
 
-func createSnapshot(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, snapID, volID volumeID) error {
-	args := []string{
-		"fs",
-		"subvolume",
-		"snapshot",
-		"create",
-		volOptions.FsName,
-		string(volID),
-		string(snapID),
-		"--group_name",
-		volOptions.SubvolumeGroup,
-		"-m", volOptions.Monitors,
-		"-c", util.CephConfigPath,
-		"-n", cephEntityClientPrefix + cr.ID,
-		"--keyfile=" + cr.KeyFile,
+func (vo *volumeOptions) createSnapshot(ctx context.Context, snapID, volID volumeID) error {
+	fsa, err := vo.conn.GetFSAdmin()
+	if err != nil {
+		util.ErrorLog(ctx, "could not get FSAdmin, can create a snapshot from %s: %s", vo.FsName, err)
+		return err
 	}
 
-	err := execCommandErr(
-		ctx,
-		"ceph",
-		args[:]...)
+	err = fsa.CreateSubVolumeSnapshot(vo.FsName, vo.SubvolumeGroup, string(volID), string(snapID))
 	if err != nil {
-		util.ErrorLog(ctx, "failed to create subvolume snapshot %s %s(%s) in fs %s", string(snapID), string(volID), err, volOptions.FsName)
+		util.ErrorLog(ctx, "failed to create subvolume snapshot %s %s(%s) in fs %s", string(snapID), string(volID), err, vo.FsName)
```

Comment:
```suggestion
util.ErrorLog(ctx, "failed to create subvolume snapshot %s %s(%s) in fs %s", string(snapID), vo.FsName,string(volID), err)
```

Reply:
It actually was this way before. I assumed `"in fs %s"` would make most sense with `%s` being the name of the filesystem?

Reply:
Yes correct, we just need to correct the error logging, can we make it as
```suggestion
util.ErrorLog(ctx, "failed to create subvolume snapshot %s %s in fs %s with error %s", string(snapID), vo.FsName,string(volID), err)
```

Reply:
@nixpanic can you move the error logging to last so that we can merge this PR?

Reply:
done!